irony-cdb-clang-complete.el

```diff
@@ -40,8 +45,15 @@
 
 (defun irony-cdb-clang-complete--locate-db ()
   (when buffer-file-name
-    (irony--awhen (locate-dominating-file buffer-file-name ".clang_complete")
-      (concat (file-name-as-directory it) ".clang_complete"))))
+    (let ((clang-complete
```

Comment:
I think it would be better to have only one "locate-dominating-file".
We should traverse the filesystem only once, and the first of the 2 files should be returned.